partiql-lang/src/main/kotlin/org/partiql/lang/eval/EvaluationSession.kt

```diff
@@ -31,7 +31,8 @@ class EvaluationSession private constructor(
     val globals: Bindings<ExprValue>,
     val parameters: List<ExprValue>,
     val context: Map<String, Any>,
-    val now: Timestamp
+    val now: Timestamp,
+    val currentUser: String?
```

Comment:
Just bouncing back ideas here: considering addition of other variables like `CURRENT_ROLE` does it make sense to add a more general argument like `sysVariables`?

Reply:
Note the `context` argument here already.  The doc comment for it appears to cover this purpose.

Said this, I, so far, continue to maintain my opinion that CURRENT_USER is best be a regular identifier, supplied through the global environment.  I do not see any functionality for it in the language, besides lookup -- which is the functionality of variables/identifiers. 

But I'll explore this PR some more before passing a firmer judgment :-)

Reply:
ACK -- the context parameter is a good point. I'll update to reflect.

As for the usage as a regular identifier, are you proposing that we modify `compileId` within `EvaluatingCompiler` to have a `when` statement to check for certain variables such as `current_user`?

I feel as if the most appropriate representation would be the function call -- as it takes in the EvaluationSession, and it wouldn't require modifying the `EvaluatingCompiler`.

Reply:
To address the current need (to be able to parse a query that mentions CURRENT_USER) all that needs to be done, I think, is removing CURRENT_USER from the list of tokens, which should make it parse as an identifier. 

If we'd like a query with CURRENT_USER to evaluate in the CLI, the best and easiest at the moment would be to augment the global bindings that CLI uses with a binding for CURRENT_USER. 

Either way, I don't foresee a need to touch EvaluatingCompiler or anything else in the processing pipeline.